ios-drag-drop.js

```diff
@@ -1,10 +1,11 @@
-(function(doc) {
+var module = (typeof module === 'object' && typeof module.exports === 'object') ? module : {};
+function _exposeIosHtml5DragDropShim(doc, config) {
```

Comment:
Can we keep keep the code in this file wrapped in an iife to protect global scope?  I love the idea of making this library commonjs-ready, but I'd love to see global scope unaffected.